tools/mkdocs.ts

```diff
@@ -0,0 +1,43 @@
+import { Command } from 'commander';
+import fs from 'fs-extra';
+import { logger } from '../lib/logger';
+import { generateDocs } from './docs';
+import { exec } from './utils/exec';
+
+process.on('unhandledRejection', (err) => {
+  // Will print "unhandledRejection err is not defined"
+  logger.error({ err }, 'unhandledRejection');
+  process.exit(-1);
+});
+
+const program = new Command('pnpm build:mkdocs')
+  .description('Build mkdocs')
+  .option('--build <boolean>', 'build docs from source', (s) =>
+    s ? s !== 'false' : undefined,
+  );
+
+void (async () => {
+  await program.parseAsync();
+  const opts = program.opts();
+  logger.info('validating docs');
+  if (opts.build) {
+    logger.info('* generate docs');
+    await generateDocs('tools/mkdocs', false);
+  } else {
+    logger.info('* using prebuild docs');
```

Comment:
The message here is not clear to me.

Are you caching a previous build maybe? To speed up a second build on the _same_ machine? Or is this about something else?

Reply:
```suggestion
    logger.info('* using prebuild docs from build step');
```
It reusing the build from here
https://github.com/renovatebot/renovate/blob/6ca0b361461edabe343452537f63761852b0f75d/.github/workflows/build.yml#L598

otherwise we get rate limited from github because of the issue search

Reply:
Your suggestion is good! Can you apply it for me?